bug1381516.kp/knowledge.md

```diff
@@ -0,0 +1,264 @@
+```
+
+
+
+
+    PythonRDD[51] at RDD at PythonRDD.scala:48
+
+
+
+
+```python
+sorted(modern_combos.map(lambda c: (c[0], 1)).countByKey().iteritems(), key=lambda x: x[1], reverse=True)
+```
+
+
+
+
+    [(u'NOTIFY_OBSERVERS_LATENCY_MS', 72463),
+     (u'DOM_SCRIPT_SRC_ENCODING', 33021),
+     (u'FX_SESSION_RESTORE_CONTENT_COLLECT_DATA_MS', 30709),
+     (u'CONTENT_LARGE_PAINT_PHASE_WEIGHT', 11613),
+     (u'IPC_WRITE_MAIN_THREAD_LATENCY_MS', 11186),
+     (u'MAIN_THREAD_RUNNABLE_MS', 7872),
+     (u'IPC_READ_MAIN_THREAD_LATENCY_MS', 6646),
+     (u'SYNC_WORKER_OPERATION', 5614),
+     (u'IPC_SYNC_RECEIVE_MS', 4227),
+     (u'IPC_MESSAGE_SIZE', 3514),
+     (u'BLOCKED_ON_PLUGIN_MODULE_INIT_MS', 2377),
+     (u'IPC_SYNC_MESSAGE_MANAGER_LATENCY_MS', 902),
+     (u'IPC_SYNC_MAIN_LATENCY_MS', 833),
+     (u'IDLE_RUNNABLE_BUDGET_OVERUSE_MS', 701),
+     (u'MESSAGE_MANAGER_MESSAGE_SIZE2', 615),
+     (u'FX_TAB_REMOTE_NAVIGATION_DELAY_MS', 433),
+     (u'CANVAS_WEBGL_FAILURE_ID', 138),
+     (u'CANVAS_WEBGL_ACCL_FAILURE_ID', 110),
+     (u'MEDIA_CODEC_USED', 20),
+     (u'IPC_SYNC_LATENCY_MS', 9),
+     (u'VIDEO_HIDDEN_PLAY_TIME_PERCENTAGE', 8),
+     (u'VIDEO_INFERRED_DECODE_SUSPEND_PERCENTAGE', 8),
+     (u'PRINT_DIALOG_OPENED_COUNT', 2)]
+
+
+
+
+```python
+sorted(modern_combos.map(lambda c: (c, 1)).countByKey().iteritems(), key=lambda x: x[1], reverse=True)[:20]
+```
+
+
+
+
+    [((u'DOM_SCRIPT_SRC_ENCODING', u'UTF-8'), 16824),
+     ((u'DOM_SCRIPT_SRC_ENCODING', u'windows-1252'), 16165),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'cycle-collector-begin'), 13727),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'garbage-collection-statistics'), 13150),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'cycle-collector-forget-skippable'),
+      12719),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'inner-window-destroyed'), 8619),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'tab-content-frameloader-created'), 7924),
+     ((u'FX_SESSION_RESTORE_CONTENT_COLLECT_DATA_MS', u'historychange'), 7537),
+     ((u'FX_SESSION_RESTORE_CONTENT_COLLECT_DATA_MS', u'pageStyle'), 7390),
+     ((u'FX_SESSION_RESTORE_CONTENT_COLLECT_DATA_MS', u'scroll'), 7389),
+     ((u'FX_SESSION_RESTORE_CONTENT_COLLECT_DATA_MS', u'storage'), 7380),
+     ((u'IPC_WRITE_MAIN_THREAD_LATENCY_MS', u'PLayerTransaction::Msg_Update'),
+      6284),
+     ((u'SYNC_WORKER_OPERATION', u'WorkerCheckAPIExposureOnMainThread'), 4926),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'content-document-global-created'), 4486),
+     ((u'IPC_SYNC_RECEIVE_MS', u'???'), 4227),
+     ((u'IPC_READ_MAIN_THREAD_LATENCY_MS', u'PCompositorBridge::Msg_DidComposite'),
+      3523),
+     ((u'NOTIFY_OBSERVERS_LATENCY_MS', u'document-element-inserted'), 3498),
+     ((u'IPC_WRITE_MAIN_THREAD_LATENCY_MS',
+       u'PCompositorBridge::Msg_PTextureConstructor'),
+      2231),
+     ((u'IPC_MESSAGE_SIZE', u'PBrowser::Msg_AsyncMessage'), 2083),
+     ((u'IPC_READ_MAIN_THREAD_LATENCY_MS', u'PBrowser::Msg_AsyncMessage'), 2031)]
+
```

Comment:
I don't really understand the conclusion here. Is it, "Yes, we've seen this recently but with different probes?"